utils/utils.py

```diff
@@ -487,37 +527,6 @@ def ordereddict_eval(str_to_eval: str):
         return str_to_eval
 
 
-def defaults_from_params(params, key=None):
```

Comment:
this was added by Blaise, right? Why remove it?

Reply:
I think this was refactored somewhere else and it became redundant